Dockerfile

```diff
@@ -52,12 +52,8 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 # Add PHP-settings
 ADD php-conf.d/ $PHP_INI_DIR/conf.d/
 
-# Create private-files volume, copy sites/default's defaults and make it a volume
-RUN mkdir private && chown -R www-data:www-data /var/www
```

Comment:
You're already doing it there https://github.com/samos123/docker-tutum-drupal/blob/master/entrypoint.sh#L90